graph/debruijnedge.h

```diff
@@ -73,9 +73,9 @@ class DeBruijnEdge
     DeBruijnNode * m_endingNode;
     GraphicsItemEdge * m_graphicsItemEdge;
     DeBruijnEdge * m_reverseComplement;
-    bool m_drawn;
-    EdgeOverlapType m_overlapType;
-    int m_overlap;
+    bool m_drawn : 1;
+    EdgeOverlapType m_overlapType : 2;
+    int m_overlap : 29;
```

Comment:
is 250M enough for overlaps?

Reply:
more than enough :) Usually it's just few symbols